docs/modeling/model-components/attributes/many-to-many.md

```diff
@@ -33,4 +53,40 @@ The name of the collection that will contain the set of objects on the other sid
 
 <Prop def="public string FarNavigationProperty { get; set; }" />
 
-The name of the navigation property on the middle entity that points at the far side of the many-to-many relationship. Use this to resolve ambiguities when the middle table of the many-to-many relationship has more than two reference navigation properties on it.
\ No newline at end of file
+The name of the navigation property on the middle entity that points at the far side of the many-to-many relationship. Use this to resolve ambiguities when the middle table of the many-to-many relationship has more than two reference navigation properties on it.
+
+``` c#
+public class Person
+{
+    public int PersonId { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+
+    [ManyToMany("Appointments", FarNavigationProperty = nameof(PersonAppointment.Appointment))]
+    public ICollection<PersonAppointment> PersonAppointments { get; set; }
+}
+
+public class Appointment
+{
+    public int AppointmentId { get; set; }
+    public DateTime AppointmentDate { get; set; }
+
+    [ManyToMany("People", FarNavigationProperty = nameof(PersonAppointment.Person))]
+    public ICollection<PersonAppointment> PersonAppointments { get; set; }
+}
+
+public class PersonAppointment
+{
+    public int PersonAppointmentId { get; set; }
+
+    public int PersonId { get; set; }
+    public Person Person { get; set; }
+
+    public int AppointmentId { get; set; }
+    public Appointment Appointment { get; set; }
+
+    // Additional reference requiring the use of FarNavigationProperty
+    public int WaiverId { get; set; }
+    public Waiver Waiver { get; set; }
```

Comment:
I like to leave out the parts that are the same as the original example for conciseness, so its clear to the user what is unique about the example:

```suggestion
public class Person
{
    ...
    
    [ManyToMany("Appointments", FarNavigationProperty = nameof(PersonAppointment.Appointment))]
    public ICollection<PersonAppointment> PersonAppointments { get; set; }
}

public class Appointment
{
    ...

    [ManyToMany("People", FarNavigationProperty = nameof(PersonAppointment.Person))]
    public ICollection<PersonAppointment> PersonAppointments { get; set; }
}

public class PersonAppointment
{
    ...

    // Adding a third reference navigation property in the middle table requires 
    // the use of FarNavigationProperty in order to resolve ambiguity.
    public int WaiverId { get; set; }
    public Waiver Waiver { get; set; }
```